process/mock/txLogsProcessorStub.go

```diff
@@ -0,0 +1,40 @@
+package mock
+
+import (
+	"github.com/ElrondNetwork/elrond-go/data"
+	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
+)
+
+// TxLogsProcessorStub -
+type TxLogsProcessorStub struct {
+	GetLogCalled func(txHash []byte) (data.LogHandler, error)
+	SaveLogCalled func(txHash []byte, tx data.TransactionHandler, vmLogs []*vmcommon.LogEntry) error
+	IsInterfaceNilCalled func() bool
+}
+
+// GetLog -
+func(txls *TxLogsProcessorStub) GetLog(txHash []byte) (data.LogHandler, error) {
+	if txls.GetLogCalled != nil {
+		return txls.GetLogCalled(txHash)
+	}
+
+	return nil, nil
+}
+
+// SaveLog -
+func(txls *TxLogsProcessorStub) SaveLog(txHash []byte, tx data.TransactionHandler, vmLogs []*vmcommon.LogEntry) error {
+	if txls.SaveLogCalled != nil {
+		return txls.SaveLogCalled(txHash, tx, vmLogs)
+	}
+
+	return nil
+}
+
+// IsInterfaceNil -
+func(txls *TxLogsProcessorStub) IsInterfaceNil() bool {
+	if txls.IsInterfaceNilCalled != nil {
```

Comment:
no need for stub function for is interface nil. return as usual return txls == nil

Reply:
done